scripts/operations/validation/ncn1.3.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#
+# MIT License
+#
+# (C) Copyright 2014-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+#Store password
+echo " Please enter Switch Password"
+read -s SSHPASS
+export SSHPASS
+
+#Update CANU
+echo "Updating CANU for network management"
+wget https://github.com/Cray-HPE/canu/releases/download/1.6.13/canu-1.6.13-1.x86_64.rpm
+rpm -Uvh canu-1.6.13-1.x86_64.rpm
+sleep 2s
+echo "--------------------------------"
+
+#Install sshpass
+echo "Install required packages"
+zypper -n install sshpass
```

Comment:
I tried this on a csm-1.3 system and it failed. The script should make sure that sshpass is actually installed before proceeding, right?